decidim-core/app/controllers/decidim/links_controller.rb

```diff
@@ -7,7 +7,7 @@ class LinksController < Decidim::ApplicationController
     skip_before_action :store_current_location
 
     helper Decidim::ExternalDomainHelper
-    helper_method :external_url
+    helper_method :external_url, :target_url
```

Comment:
This change does not seem necessary.
```suggestion
    helper_method :external_url
```